sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -309,6 +309,18 @@ functionCall
     | relevanceFunction                                             #relevanceFunctionCall
     | highlightFunction                                             #highlightFunctionCall
     | positionFunction                                              #positionFunctionCall
+    | GET_FORMAT LR_BRACKET getFormatArgs RR_BRACKET                #getFormatFunctionCall
+    ;
+
+getFormatArgs
+    : getFormatType COMMA functionArg
```

Comment:
Maybe you can do something like this to have the common style:
```suggestion
    | getFormatFunction                                             #getFormatFunctionCall
    ;

getFormatFunction
    : GET_FORMAT LR_BRACKET getFormatType COMMA functionArg RR_BRACKET
```

Maybe you need to update AstBuilder to make this working

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/210/commits/c804a89caee97c1817142e7491cb9ee54d64550e